doc/en/doctest.rst

```diff
@@ -11,6 +11,17 @@ can change the pattern by issuing::
 on the command line. Since version ``2.9``, ``--doctest-glob``
 can be given multiple times in the command-line.
 
+You can specify the encoding that will be used for those doctest files
+using the ``doctest_encoding`` ini option:
```

Comment:
One last thing, please add this:

```rst
You can specify the encoding that will be used for those doctest files
using the ``doctest_encoding`` ini option:

.. versionadded:: 3.1

.. code-block:: ini
```

To convey to the users that this is new and was added in 3.1.